pint/facets/numpy/numpy_func.py

```diff
@@ -741,21 +741,23 @@ def _base_unit_if_needed(a):
 
 
 @implements("trapz", "function")
```

Comment:
not sure what the policy on deprecations is, but maybe add a comment about removing this once `numpy>=2` is the minimum supported version? That usually makes it much more likely to get cleaned up.

Reply:
Sounds good.